index.html

```diff
@@ -14,21 +14,115 @@
+          <h1>Tonic</h1>
+          <ul class="tech">
+            <li class="company-name"><a>company</a></li>
+            <li><a href="#"><img src="images/Counter.png" alt="dot"></a></li>
+            <li><a>Back End Dev</a></li>
+            <li><a href="#"><img src="images/Counter.png" alt="dot"></a></li>
+            <li><a>2015</a></li>
+          </ul>
+          <P>A daily selection of privately personalized reads; no accounts or sign-ups required.</P>
+          <ul class="prog-languages">
+            <li class="lang">html</li>
+            <li class="lang">css</li>
+            <li class="lang">javaScript</li>
+          </ul>
+          <button type='button'>See Project</button>
+        </section>
+
+        <section class="project">
+          <img src="./images/Snapshoot-Portfolio4.png" alt="snapshot1">
+          <h1>multi-post stories</h1>
+          <ul class="tech">
+            <li class="company-name"><a>company</a></li>
+            <li><a href="#"><img src="images/Counter.png" alt="dot"></a></li>
+            <li><a>Back End Dev</a></li>
+            <li><a href="#"><img src="images/Counter.png" alt="dot"></a></li>
+            <li><a>2015</a></li>
+          </ul>
+          <P>A daily selection of privately personalized reads; no accounts or sign-ups required.</P>
+          <ul class="prog-languages">
+            <li class="lang">html</li>
+            <li class="lang">css</li>
+            <li class="lang">javaScript</li>
+          </ul>
+          <button type='button'>See Project</button>
+        </section>
+      </section>
+
+      <section id="about-me">
+        <h1>About Myself</h1>
+        <p>Hello I’m a software developer! I can help you build a product,
+           feature or website Look through some of my work and experience! 
+           If you like what you see and have a project you need coded, 
+           don’t hestiate to contact me.
+        </p>
+
+        <h3>let's connect</h3>
+        <ul class="social">
+          <li><img src="./images/Vector.png" alt="twitter"></li>
+          <li><img src="./images/Linkedin-icon.jpg" alt="linkedin"></li>
+          <li><img src="./images/Vector.jpg" alt="email"></li>
+          <li><img src="./images/Vector-2.jpg" alt="github"></li>
+          <li><img src="./images/Vector-3.jpg" alt="angel-list"></li>
+        </ul>
+
+        <button type='button'>Get my resume</button>
+
+        <ul id="list">
+          <li class="expertise">
+            <h5>Languages</h5><img src="./images/Union.png" alt="expand" class="arrow"></li>
+            <li><ul id="languages-list">
+              <li class="languages"><h6>JavaScipt</h6><img src="./images/Ellipse1.png" alt="javascript logo"></li>            
+              <li class="languages"><h6>HTML</h6><img src="./images/Ellipse2.png" alt="html logo"></li>            
+              <li class="languages"><h6>CSS</h6><img src="./images/Ellipse3.png" alt="css logo"></li>
```

Comment:
- [ ] [REQUIRED] please, make sure that the languages section is aligned according to the Figma design 🖼️.

your design | Figma Design
-------|---------
![image](https://user-images.githubusercontent.com/60979926/164736760-75da809a-60d3-488e-be3b-068bc74d5851.png) | ![image](https://user-images.githubusercontent.com/60979926/164737088-1b301ebe-cd4b-43c0-ad81-ee33f96361af.png)